testservice/src/main/kotlin/com/wire/kalium/testservice/managed/InstanceService.kt

```diff
@@ -222,33 +221,47 @@ class InstanceService(
                             model = instanceRequest.deviceName
                         )
                     )) {
-                        is RegisterClientResult.Failure ->
-                            throw WebApplicationException("Instance $instanceId: Client registration failed")
-
                         is RegisterClientResult.Success -> {
-                            clientId = result.client.id.value
+                            val clientId = result.client.id.value
                             log.info("Instance $instanceId: Device $clientId successfully registered")
-                            syncManager.waitUntilLive()
+
+                            val instance = Instance(
+                                instanceRequest.backend,
+                                clientId,
+                                instanceId,
+                                instanceRequest.name,
+                                coreLogic,
+                                instancePath,
+                                instanceRequest.password,
+                                System.currentTimeMillis() - before,
+                                System.currentTimeMillis()
```

Comment:
suggestion: could you please define these parameters as variables then it would be clear what are these and why we have them like this?

Reply:
Thanks. I commited a small fix.